core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -376,7 +366,7 @@ public void shutdown() {
         scheduler.shutdown();
         for (ExecutorHolder executor : executors.values()) {
             if (executor.executor() instanceof ThreadPoolExecutor) {
-                ((ThreadPoolExecutor) executor.executor()).shutdown();
+                executor.executor().shutdown();
```

Comment:
These are cosmetic changes, let's revert them here.

Reply:
this looks like a good change to me though. It resolves a warning in my IDE caused by the unnecessary cast. Why not keep it?

Reply:
It’s fine @javanna, pull it in in this one. I would prefer we keep changes like this out of PRs, they can go in separately, I want less to think about when I review.